core/src/test/java/com/klaytn/caver/feature/KIP7Test.java

```diff
@@ -0,0 +1,1252 @@
+            allowance = tokenHandler_owner.allowance(ownerAddress, spenderAddress).send();
+            assertEquals(allowance, burnAmount);
+
+            KlayTransactionReceipt.TransactionReceipt receipt = tokenHandler_spender.burnFrom(ownerAddress, burnAmount).send();
+            BigInteger balance = tokenHandler_owner.balanceOf(ownerAddress).send();
+            BigInteger totalSupply = tokenHandler_owner.totalSupply().send();
+            assertEquals(preBalance.subtract(burnAmount), balance);
+            assertEquals(preTotalSupply.subtract(burnAmount), totalSupply);
+
+            ContractInitialSupply = totalSupply;
+
+        } catch (Exception e) {
+            e.printStackTrace();
+            fail();
+        }
+    }
+
+    //KCT-026
+    @Test
+    public void supportsInterface() {
+        final String INTERFACE_ID_KIP13 = "0x01ffc9a7";
+        final String INTERFACE_ID_KIP7_PAUSABLE = "0x4d5507ff";
+        final String INTERFACE_ID_KIP7_BURNABLE = "0x3b5a0bf8";
+        final String INTERFACE_ID_KIP7_MINTABLE = "0xeab83e20";
+        final String INTERFACE_ID_KIP7_METADATA  = "0xa219a025";
+        final String INTERFACE_ID_KIP7  = "0x65787371";
+        final String INTERFACE_ID_FALSE = "0xFFFFFFFF";
+
+        try {
+            KIP7 tokenHandler_owner = KIP7.load(mContractAddress, mCaver, mDeployerTxManager, new DefaultGasProvider());
+
+            boolean isSupported_KIP13 = tokenHandler_owner.supportsInterface(Numeric.hexStringToByteArray(INTERFACE_ID_KIP13)).send();
+            assertTrue(isSupported_KIP13);
+
+            boolean isSupported_KIP7_PAUSABLE = tokenHandler_owner.supportsInterface(Numeric.hexStringToByteArray(INTERFACE_ID_KIP7_PAUSABLE)).send();
+            assertTrue(isSupported_KIP7_PAUSABLE);
+
+            boolean isSupported_KIP7_BURNABLE = tokenHandler_owner.supportsInterface(Numeric.hexStringToByteArray(INTERFACE_ID_KIP7_BURNABLE)).send();
+            assertTrue(isSupported_KIP7_BURNABLE);
+
+            boolean isSupported_KIP7_MINTABLE = tokenHandler_owner.supportsInterface(Numeric.hexStringToByteArray(INTERFACE_ID_KIP7_MINTABLE)).send();
+            assertTrue(isSupported_KIP7_MINTABLE);
+
+            boolean isSupported_KIP7_METADATA = tokenHandler_owner.supportsInterface(Numeric.hexStringToByteArray(INTERFACE_ID_KIP7_METADATA)).send();
+            assertTrue(isSupported_KIP7_METADATA);
+
+            boolean isSupported_KIP7 = tokenHandler_owner.supportsInterface(Numeric.hexStringToByteArray(INTERFACE_ID_KIP7)).send();
+            assertTrue(isSupported_KIP7);
+
+            boolean isSupported_FALSE = tokenHandler_owner.supportsInterface(Numeric.hexStringToByteArray(INTERFACE_ID_FALSE)).send();
+            assertFalse(isSupported_FALSE);
+        } catch (Exception e) {
+            e.printStackTrace();
+            fail();
+        }
+    }
+
+    //KCT-028
+    @Test
+    public void getTransferEvent_Burn() {
```

Comment:
How about making KlaytTransactionReceipt object to include multiple events for testing get***Event?

You can check the event after actually initiating the transaction at the place where you test the method. (For now test code, some check, but some not)

However, the getTransferEvent method only needs to check the operation of the Transfer event regardless of the method.
I don't think it is necessary to test for each method.